cfpb/cfgov-refresh
# Ignore `partial` param in pagination links

Fixes [GHE#104](https://GHE/eregs/regulations-3000/issues/104) by not adding the `partial` parameter to pagination links.

## Testing

1. Visit a [Regs3k search results page](http://localhost:8000/policy-compliance/rulemaking/regulations/search-regulations/results/?q=disclosure)
1. Scroll to the bottom and click the `Next >` button in the pagination component.
1. The browser should load page 2 of the search results (instead of showing a broken-looking partial search results page).


## Checklist
